### ISSUE TITLE
Preferred Single Quote doesn't allow me to use double quotes, even in content

### ISSUE THREAD

**Issue by felipenmoura** [rossPatton/stylint#271]
Hi.
I am using the stylint with Preferred Single Quote enabled.
But it fails on this code:

```css
seletor:after {
    content: '"'; // " between '
}
```

It should allow me to use in that situation.
It still blocks it even if I scape it with \", and using `&quote;` will not work either.

What am I supposed to do? Disable this validation?!

Thanks.

**Comment by rossPatton** [comment]
you're right, that SHOULD work, but it's possible the regex for that test doesn't take into account empty strings like that.

in the meantime, until it's fixed, there are annotations you can use to disable errors

```stylus
seletor:after {
    content: '"'; // @stylint ignore
}
```

**Comment by rossPatton** [comment]
this very specific case should be fixed in 1.5.1
